### ISSUE TITLE
File autocomplete: Lacks keyboard navigation & ARIA

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#11322]
Our custom file autocompletion widget lacks keyboard navigation and screen reader support.

There's an interesting, accessible demo at https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html — We'd want to specifically look at [example 3](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html#ex3_label).

It probably would involve a bit of retrofitting and adapting, not a complete reimplementation.

**Comment by garrett** [comment]
Closing as a duplicate of https://github.com/cockpit-project/cockpit/issues/11300
